packages/blade/src/components/Accordion/_decisions/decisions.md

```diff
@@ -66,7 +66,7 @@ import { Accordion, AccordionItem } from '@razorpay/blade';
 | Key         | Type            | Default     | Description                                                                | Required |
 | ----------- | --------------- | ----------- | -------------------------------------------------------------------------- | -------- |
 | title       | `string`        | `undefined` | Title text content                                                         | ✅       |
-| description | `string`        | `undefined` | Body text content                                                          | ✅       |
+| description | `string`        | `undefined` | Body text content                                                          |          |
```

Comment:
i don't know but i feel we shall remove description as a prop when we have children already. the initial idea was to have description always but now that's gone based on one of the use cases of X i feel we shall remove description all together 😞

Reply:
I think it would be beneficial to keep it:

- Majority of accordion items will have only description text (more convenient DX)
- Otherwise it'll be cumbersome for consumers to always wrap the passed text with correct props:

```jsx
<AccordionItem title="Title">
  <Text type="subtle">Description</Text> // <-- In most cases users will have to wrap this and remember type="subtle"
</AccordionItem>
```

Reply:
which should be fine. the persona of users you're talking about doesn't exist in our teams 🤣

```
<AccordionItem title="Title">
  <Text type="subtle">Description</Text> // <-- In most cases users will have to wrap this and remember type="subtle"
</AccordionItem>
```
I see this only being used majority of the times